app/middleware/cache_headers.py

```diff
@@ -26,9 +26,12 @@ def __call__(self, request):
 
         # Code to be executed for each request/response after
         # the view is called.
-
-        if request.path in ['/checker', '/get-token']:
-            # never cache the /checker and /get-token endpoints.
+        # comparing the last separation by / in both strings
+        if request.path.split('/')[-1] in [
+            i.split('/')[-1] for i in
+            ['/checker', f'{settings.API_BASE}/get-token', f'/{settings.HEALTHCHECK_ENDPOINT}']
+        ]:
+            # never cache the /checker, /get-token and healthcheck endpoints.
```

Comment:
We could simplify this by choosing patterns:
`/XYZ`: is used cluster-internally (not publicly visible), currently `/checker` and `/metrics`, uncached
`/api/XYZ` and `/api/stac/XYZ`: is used/visible externally, uncached, e.g. `/api/healthcheck`, `/api/stac/get-token`, ...
`/api/stac/static`: special handling of cache settings by whitenoise
`/api/stac/v0.9*: default cache settings

This would require some changes in `app/config/urls.py` and `app/stac_api/urls.py` (essentially moving the `/api/stac/` prefix part from `app/stac_api/urls.py` to `app/config/urls.py`...

Reply:
Good idea to resume we would have the following patterns:

- '/\w+' => cluster-internals, no cache; /checker, /metrics
- '/api/stac/\w+' => visible external, no cache; /api/stac/healthcheck /api/stac/get-tocken
- '/api/stac/admin/.*' => cache handled by django admin site
- '/api/stac/static/.*' => cache handled by Whitenoise
- '/api/stac/v0.9/.*' => add cache headers